.gitignore

```diff
@@ -151,3 +151,5 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
+#ds_store ignore
+**/.DS_Store
```

Comment:
This should go in [`$XDG_CONFIG_HOME/git/ignore`](https://git-scm.com/docs/gitignore)

Reply:
We have made this change. Removed the DS_Store files from gitignore.